### DISCUSSION TITLE
Setting metadata for GitHub PullRequest

### DISCUSSION THREAD

**Discussion by lindhe** [renovatebot/renovate#29594] [Request Help] [answered]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

Renovate v37.399.10 running on Kubernetes working towards GitHub Enterprise

### Please tell us more about your question or problem

I'd like to set some default metadata attributes for PRs created by Renovate, like Labels and Project in GitHub. Is that possible?

**Comment by rarkins** [comment, marked as answer]
Labels Yes: [Renovate docs, configuration options `labels`](https://docs.renovatebot.com/configuration-options/#labels)

Projects No, see:

- https://github.com/renovatebot/renovate/issues/3616

**Reply by lindhe** [reply]
Thank you! 👍 I was looking for it among `prBodyColumns`, `prBodyDefinitions`, `prTitle`, etc, which was why I missed `labels` in the docs.
